packages/env/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### Breaking Change
+
+-   Removed the second shell from the execution of `run` commands. This removes the need for double-escaping and fixes problems with undocumented evaluation of escaped shell tokens before they are passed to the container. As a consequence of this change, quoted strings will now be treated as single arguments instead of separate ones. For example, `npx wp-env run cli "wp help"` will attempt to run `"wp help"` on the container instead of `wp` with `help` as an argument. If you are using commands with excess escaping or quotes you will need to review them and ensure they are compatible with the update. 
```

Comment:
I might rework this help text to focus more on how it impacts the consumer, especially since the second shell stuff is an internal implementation detail. I'm not sure exactly how, but maybe some examples would help:

Each of these does the same thing as before:
```sh
$ npx wp-env run cli echo "hello world"
# echo hello world
$ npx wp-env run cli echo hello world
# echo hello world
$ npx wp-env run cli echo \"hello world\"
# echo "hello world"
```

This breaks now, but previously executed `echo hello world`
```
npx wp-env run cli "echo hello world"
```

Reply:
This is good feedback, I've gone ahead and pushed a change to the documentation around this.